### PR TITLE
[AmazonEchoControl] Fix NPE #8885 

### PR DESCRIPTION
**Pull request by CSchlipp**
[AmazonEchoControl] Fix NPE #8885 

Fixed obviously wrong boolean condition & ran mvn spotless apply

Binding is starting successfully again, Alexa control in terms of TTS, etc. is working.
Due to limited possibilities, controlling SmartHome devices connected to Alexa has not been tested.
However, the previous boolean condition did not make sense this way either, so I do not expect a regression...

### CODE REVIEW

**Comment by J-N-K** [comment]
Same fix as #8888 in main

File: bundles/org.openhab.binding.amazonechocontrol/pom.xml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (J-N-K) [inline bundles/org.openhab.binding.amazonechocontrol/pom.xml:3]: Please revert this changes, it breaks the build

**Review by J-N-K — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.amazonechocontrol/pom.xml (hunk 1 above)
Comment (CSchlipp) [reply bundles/org.openhab.binding.amazonechocontrol/pom.xml]: Strange - this was the result of mvn spotless apply...
Reverted it with the last commit

**Comment by CSchlipp** [comment]
Sorry, didn't notice that there was another PR in the meantime. It wasn't liked to this issue...

After reverting the changes as requested, Travis build failed at mvn spotless check.

**Comment by J-N-K** [comment]
The issue wasn't linked because I didn't see that issue. I was trying to fix a REST issue in OH3...

Interesting that it fails. Because I requested that change because travis failed before with the spotless check on the same file.

**Comment by J-N-K** [comment]
The formatting is broken in all POMs due to the changes by the unleash plugin.Since this change is known to work on the 3.0 branch, I'll merge it anyway.

**Review by J-N-K — APPROVED** [review]
